compiler/qsc_frontend/src/compile.rs

```diff
@@ -40,60 +40,33 @@ use thiserror::Error;
 bitflags! {
     #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
     pub struct RuntimeCapabilityFlags: u32 {
```

Comment:
I think the name "target capability" would start to make more sense here, since it includes arbitrary  target hardware capabilities like Qubit Reset. It also sounds nicely similar to "target profile", which would be a group of "target capabilities".